pymc/model.py

```diff
@@ -478,10 +481,10 @@ def __init__(self, mean=0, sigma=1, name=''):
 
     if TYPE_CHECKING:
```

Comment:
is this even necessary?

Reply:
Yes - I tried removing it at first, but pyright is unable to infer that `Model`, which is a child of the metaclass `ContextMeta`, implements `__enter__`.